.cirrus.yml

```diff
@@ -33,7 +44,7 @@ test_task:
         # Linux, so losing this part of the data makes very little difference.
         SKIP_UPLOAD_COVERAGE: 1
   go_version_script: go version
-  test_script: go test -race ./...
+  test_script: go test $TEST_FALG ./...
```

Comment:
Typo here and above: s/TEST_FALG/TEST_FLAG/. You're also replacing `-race` with `-trace` which appears to be another typo.